README.md

```diff
@@ -296,6 +296,20 @@ Some constants are available in the config.json file to customize your graphical
 - `screen_width`: width of the game window. 
 - `screen_height`: height of the game window.
 - `leaderboard_width`: width of the leaderboard.
+- `add_suffix_to_nickname`: whether to add a random suffix to the nickname.
+
+### Simulate Multiple Clients
```

Comment:
I don't think students need additional complexity. They can already create a server with lots of randomly selected agents.